packages/core/number/src/number.ts

```diff
@@ -6,4 +6,13 @@ function wrap(index: number, max: number) {
   return (max + index) % max;
 }
 
-export { clamp, wrap };
+// https://github.com/tmcw-up-for-adoption/simple-linear-scale/blob/master/index.js
```

Comment:
@benoitgrelard we spoke about moving things when they're used 3+ times but I had already done this before that discussion and it feels like its a bit too complex to be compying everywhere to me. Thoughts on leaving this here?